nifi-nar-bundles/nifi-smb-bundle/nifi-smb-processors/src/main/java/org/apache/nifi/processors/smb/ListSmb.java

```diff
@@ -79,32 +90,31 @@
         "previous node left off without duplicating all of the data.")
 @InputRequirement(Requirement.INPUT_FORBIDDEN)
 @WritesAttributes({
-        @WritesAttribute(attribute = "filename", description = "The name of the file that was read from filesystem."),
-        @WritesAttribute(attribute = "shortname", description = "The short name of the file that was read from filesystem."),
-        @WritesAttribute(attribute = "path", description =
+        @WritesAttribute(attribute = FILENAME, description = "The name of the file that was read from filesystem."),
+        @WritesAttribute(attribute = SHORT_NAME, description = "The short name of the file that was read from filesystem."),
+        @WritesAttribute(attribute = PATH, description =
                 "The path is set to the relative path of the file's directory "
                         + "on filesystem compared to the Share and Input Directory properties and the configured host "
                         + "and port inherited from the configured connection pool controller service. For example, for "
                         + "a given remote location smb://HOSTNAME:PORT/SHARE/DIRECTORY, and a file is being listed from "
                         + "smb://HOSTNAME:PORT/SHARE/DIRECTORY/sub/folder/file then the path attribute will be set to \"sub/folder/file\"."),
-        @WritesAttribute(attribute = "absolute.path", description =
+        @WritesAttribute(attribute = ABSOLUTE_PATH, description =
                 "The absolute.path is set to the absolute path of the file's directory on the remote location. For example, "
                         + "given a remote location smb://HOSTNAME:PORT/SHARE/DIRECTORY, and a file is being listen from "
                         + "SHARE/DIRECTORY/sub/folder/file then the absolute.path attribute will be set to "
                         + "SHARE/DIRECTORY/sub/folder/file."),
```

Comment:
When I run ListSmb, `absolute.path` attribute does not contain the SHARE part. I think this is the correct behaviour and the documentation needs to be corrected.